bin/svn-git-up

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# Update Git and SVN repos checked out in the same directory.
+#
+# Pass a branch name like 4.6 as the sole argument to checkout that branch. Otherwise, master/trunk is checked out.
+
+cd $(dirname "$0")/../public/core-git
```

Comment:
I think we should rather remove this line. Other commands can also be run from any directory and will apply to the directory you're in. While I understand the idea behind this, I think it's confusing in relation to other commands.

Also, while it's likely that the script is run from that directory, there's still the chance someone might run it elsewhere.